src/operations/blas2/trsv.hpp

```diff
@@ -0,0 +1,142 @@
+/***************************************************************************
+ *
+ *  @license
+ *  Copyright (C) Codeplay Software Limited
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  For your convenience, a copy of the License has been included in this
+ *  repository.
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *  SYCL-BLAS: BLAS implementation using SYCL
+ *
+ *  @filename trsv.hpp
+ *
+ **************************************************************************/
+
+#ifndef TBSV_HPP
+#define TBSV_HPP
+#include "operations/blas2_trees.h"
+#include "operations/blas_operators.hpp"
+#include "views/view_sycl.hpp"
+#include <stdexcept>
+#include <vector>
+namespace blas {
+
+/**
+ * @struct Trsv
+ * @brief Tree node representing a triangular band matrix_ vector_
+ * multiplication.
+ */
+template <typename lhs_t, typename matrix_t, typename vector_t,
+          uint32_t local_range, bool is_upper, bool is_transposed,
+          bool is_unitdiag>
+SYCL_BLAS_INLINE
+Trsv<lhs_t, matrix_t, vector_t, local_range, is_upper, is_transposed,
+     is_unitdiag>::Trsv(lhs_t &_l, matrix_t &_matrix,
+                        typename Trsv<lhs_t, matrix_t, vector_t, local_range,
```

Comment:
The formatting seems weird here, have you ran `clang-format` on this file?

Reply:
Yes, I double-checked.